### DISCUSSION TITLE
Which version of pnpm is used by Renovate?

### DISCUSSION THREAD

**Discussion by Jolg42** [renovatebot/renovate#12214] [Request Help] [answered]
I'm curious to know if Renovate is regularly bumping the pnpm dependency for example and which version it's running now.

Also if I want to always run on the latest version would setting this like this work?
```
{
  "constraints": {
    "pnpm": "latest"
  }
}
```
Found the constraints feature in https://docs.renovatebot.com/configuration-options/#constraints

**Comment by rarkins** [comment, marked as answer]
How are you running Renovate?

**Reply by Jolg42** [reply]
Oh yeah forgot to mention, it's the hosted version we use for https://github.com/prisma/prisma

**Reply by rarkins** [reply]
npm, Yarn and pnpm should all be installed dynamically in the hosted app, based off `engines` values in `package.json`.

**Reply by Jolg42** [reply]
So if nothing is specified in either `engines` (package.json) or `constraints` (renovate config) is the latest version used?

I'll add the requirement in `engines`, thanks for the suggestion!

**Reply by viceice** [reply]
> So if nothing is specified in either `engines` (package.json) or `constraints` (renovate config) is the latest version used?

yes
